src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -118,7 +118,10 @@ functional_modules::ProcessResult VRModule::ProcessMessage(
   }
 
   if (!IsVRServiceSupported()) {
-    SendUnsupportedServiceResponse(msg);
```

Comment:
I want to make a send response common function. In the future we can reuse it in case we have to send other responses